rita_exit/src/network_endpoints/mod.rs

```diff
@@ -191,119 +188,49 @@ pub async fn get_exit_timestamp_http(_req: HttpRequest) -> HttpResponse {
 
 /// This function takes a list of exit ips in the cluster from the exit registration smart
 /// contract, and returns a list of exit ips that are in the same region and currency as the client
-/// if this exit fits the region and currenty requirements it will always return a list containing itself
-/// even if this exit is not in the smart contract. If a client is speaking with this exit then the exit
-/// data is in the config and this is considered to be a key exchange in and of itself.
-pub async fn get_exit_list(request: Json<EncryptedExitClientIdentity>) -> HttpResponse {
-    let exit_settings = get_rita_exit();
-    let our_secretkey: WgKey = exit_settings.network.wg_private_key.unwrap();
-    let our_secretkey = our_secretkey.into();
-
-    let their_nacl_pubkey = request.pubkey.into();
-
-    let contact = Web3::new(&get_web3_server(), CLIENT_STATUS_TIMEOUT);
+/// This exit may not be included in the list returned by the smart contract! If this is the case it must
+/// first be added to the root of trust server, and clients will know to choose a different exit.
```

Comment:
This comment needs some more detail to avoid confusion for future readers IMO.

Because exits are running their get_exit_list endpoint on a webserver with a multihomed ip within the babel mesh any exit that is plugged in and online will be getting exit server list requests, regardless of if that exit is on the list or otherwise. This is secure due to the signatures of course, but it's important to realize that clients could talk to any one of an arbitrary number of exits on the network for this list and then find only a very small subset, or even none of those exits are permitted roaming targets once it verifies the list results.